### PR TITLE
docs: update x/foundation specification

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch will update the documentation on the `x/foundation` specification.
TODO: Update the references in README.md

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [x] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by 0Tech** [comment]
Blocked by #704.

File: x/foundation/README.md [hunk 1]
```diff
@@ -0,0 +1,1534 @@
...
+
+# Concepts
+
+## Foundation Operator
+
+A foundation operator is a module account associated with the foundation and a
+decision policy. It is an "administrator" which has the ability to add, remove
+and update members in the foundation.
+`x/foundation` has several messages which cannot be triggered but by the
+operator. It includes membership management messages, and other messages which
```
Comment (zemyblue) [inline x/foundation/README.md:59]:
```
`x/foundation` has several messages which cannot be triggered but by the
 operator.
```
Does It means there is several message only operator can control? I think it is a little difficult. :)

File: x/foundation/README.md [hunk 2]
```diff
@@ -0,0 +1,1534 @@
...
+  foundation_version: "1"
+  id: "1"
+  messages:
+  - '@type': /lbm.foundation.v1.MsgWithdrawFromTreasury
+    amount:
+    - amount: "1000000000"
+      denom: stake
+    operator: link1...
+    to: link1...
+  metadata: show-me-the-money
```
Comment (zemyblue) [inline x/foundation/README.md:760]: `show-me-the-money` 😆

**Review by zemyblue — COMMENTED** [review]
There is not the description of `GovMint`.

**Comment by 0Tech** [comment]
> There is not the description of `GovMint`.

I don't have enough information about the _exact_ specification of `GovMint`. So I left it out. I may contact to the planning team later.

File: x/foundation/README.md (hunk 1 above)
Comment (0Tech) [reply x/foundation/README.md:59]: Hmm, I don't see any problem. Our tech docs team approved (i.e. didn't modified) this sentence and AFAIK the official docs would use it as is.

**Review by dudong2 — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Comment by 0Tech** [comment]
Waiting for the PR #729 merged.

**Pull request merged**